2.Firmware/HoloCubic-fw/src/sd_card.cpp

```diff
@@ -5,6 +5,7 @@ void SdCard::init()
 {
 
 	SPIClass* sd_spi = new SPIClass(HSPI); // another SPI
+	sd_spi->begin(14, 26, 13, 15); // Replace default HSPI pins
```

Comment:
Prefert add macro like `#define SD_SCL XX` on top of file.

Reply:
What's more, remove the related description about modifing library in README.md.